bullet/src/SimulationFeatures.hh

```diff
@@ -40,6 +42,11 @@ class SimulationFeatures :
       ForwardStep::Output &_h,
       ForwardStep::State &_x,
       const ForwardStep::Input &_u) override;
+
+  public: std::vector<ContactInternal> GetContactsFromLastStep(
+      const Identity &/* _worldID */) const override
+      { // TODO(lobotuerk): Implement contacts getter, could be like https://pybullet.org/Bullet/phpBB3/viewtopic.php?t=2855
```

Comment:
If this is a Dummy implementation, I suggest printing to ignwrn something to be aware when it's used JIC

Reply:
Updated on 30ae58a